ai/STA/Tactic/receive_pass.py

```diff
@@ -86,7 +86,7 @@ def wait_for_ball(self):
             self.next_state = self.wait_for_ball
             return CmdBuilder().build()
         orientation = (self.game_state.ball_position - self.player.position).angle
-        return CmdBuilder().addMoveTo(Pose(self.player.position + latteral_move.view(Position), orientation),
```

Comment:
I don't understand that one, the error trace said that `latteral_move` is of type Position, not np.array

Reply:
t'as tu tester ca en sim ou real-life?

Reply:
sim

Reply:
Btw je sais si j'étais claire: les lignes que j'ai changer fix le problème. Je comprends juste pas pourquoi cette ligne ne marche plus.